test.js

```diff
@@ -0,0 +1,73 @@
+const fs = require('fs');
+const exec = require('child_process').exec;
+const Ajv2020 = require("ajv/dist/2020"); // see: https://ajv.js.org/json-schema.html#draft-2020-12
+const addFormats = require("ajv-formats");
+
+// Default schema to be used if no file-specific one is defined
+const DEFAULT_SCHEMA = 'DCC.ValueSets.schema.json';
+
+// These files will be excluded from the schema check
+const FILES_TO_EXCLUDE = [
+    'package.json',
+    'package-lock.json'
+];
+
+// Init ajv
+const ajv = new Ajv2020({
+    strict: false
```

Comment:
Why non-strict? Options that I regularly use:
```javascript
const ajv = new Ajv({
    allErrors: true,        // don't stop after 1st error
    strict: true,
    allowUnionTypes: true,  // necessary for validating against the meta-schema
    validateSchema: false   // prevent that Ajv throws with 'no schema with key or ref "https://json-schema.org/draft/2020-12/schema"'
    // It seems weird that Ajv can't validate the schemas themselves (against a meta-schema) but you have to tweak things to do that yourself...
})
```

Reply:
I'm using the same settings as in the original version.

Reply:
That's a good thing. We could entertain adding `allErrors: true` to avoid that people (feel they have to) round-trip through a GitHub Action showing at most 1 error (per valueset).